notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>Add heading</p><p> </p><p>For more details, please go through this <a href="https://cloud.google.com/python/docs/reference/aiplatform/latest/google.cloud.aiplatform.AutoMLTabularTrainingJob" target="_blank">documentation for <strong>AutoMLTabularTrainingJob</strong> Class</a>. -&gt;</p><p>Learn more about [Class AutoMLTabularTrainingJob](http://....)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>updated.</p>